Godeps/_workspace/src/k8s.io/kubernetes/pkg/client/unversioned/clientcmd/validation.go

```diff
@@ -143,6 +157,10 @@ func ConfirmUsable(config clientcmdapi.Config, passedContextName string) error {
 func validateClusterInfo(clusterName string, clusterInfo clientcmdapi.Cluster) []error {
 	validationErrors := make([]error, 0)
 
+	if reflect.DeepEqual(clientcmdapi.Cluster{}, clusterInfo) {
```

Comment:
A nil map would still be someone setup the config.  I will add a test.

On Thu, Nov 5, 2015 at 11:00 AM, David Eads <notifications@github.com>
wrote:

> In
> Godeps/_workspace/src/k8s.io/kubernetes/pkg/client/unversioned/clientcmd/validation.go
> <https://github.com/openshift/origin/pull/5722#discussion_r44028845>:
>
> > @@ -143,6 +157,10 @@ func ConfirmUsable(config clientcmdapi.Config, passedContextName string) error {
> >  func validateClusterInfo(clusterName string, clusterInfo clientcmdapi.Cluster) []error {
> >  	validationErrors := make([]error, 0)
> >
> > +	if reflect.DeepEqual(clientcmdapi.Cluster{}, clusterInfo) {
>
> I would expect you have to have problems with nil maps versus empty maps.
> If you don't have an issue now, be sure to add a test that goes through the
> load method. I remember a pull upstream that was looking at having empty
> maps instead of nil ones.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/5722/files#r44028845>.
>